styles/styles.css

```
@@ -0,0 +1,347 @@
+Grijs #7A7A7A (voor de service-tegel tekst);*/
+/*//////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+.container-section1 a:link {
+    color: #5722CD;
+    text-decoration: none;
+}
+
+section {
+    display: flex;
+    justify-content: center;
+}
+
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700&family=Roboto:wght@300&display=swap');
+h1 h2 h3 h4 h5 {
+    font-family: 'Merriweather', serif;
+}
+
+p {
+    font-family: 'Roboto', sans-serif;
+    font-size: small;
+}
+
+h5 {
+    color: #5722CD;
+    margin-top: 5px;
+    margin-bottom: 5px;
+}
+
+.container2-section1 p {
+    color: black;
+    margin-top: 20px;
+}
+
+.container-section1 h1 {
+    max-width: 500px;
+}
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+.squiggle-line img {
+    max-width: 30px;
+    margin-bottom: 20px;
+    margin-top: 20px;
+}
+
+
+.squiggle {
+    position: absolute;
+    top: 90%;
+    right: 48%;
+}
+
+.icon {
+    height: 50px;
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+
+/* --------------- 4.1 Header ------------------------- */
+
+header {
+    display: flex;
+    flex-direction: column;
+    background: rgb(72, 69, 228);
+    background: linear-gradient(90deg, rgba(72, 69, 228, 1) 0%, rgba(80, 50, 216, 1) 50%, rgba(87, 34, 205, 1) 100%);
+    color: white;
+    flex-grow: 1;
+    flex-shrink: 1;
+    position: relative;
+}
+
+nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: 100%;
+}
+
+ul {
+    display: flex;
+    column-gap: 40px;
+    list-style: none;
+    align-items: center;
+    margin: 10px;
+}
+
+nav a:link {
+    color: white;
+    text-decoration: none;
+}
+
+nav a:visited {
+    color: white;
+}
+
+button {
+    background-color: #ED4385;
+    border-radius: 25px;
+    color: white;
+    padding: 10px;
+    padding-inline: 20px;
+    text-transform: uppercase;
+    font-size: 8px;
+}
+
+
+/* --------------- 4.2 Introduction ------------------- */
+
+
+.container-section1 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    margin: 50px 100px 50px 100px;
```

Dit kun je korter schrijven:
```suggestion
    margin: 50px 100px;
```